mood-analysis.rb

```diff
@@ -19,6 +24,15 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(sentence)
+  sentence = sentence.gsub(/[!.,#]/, "")
+end
+
+
+def happy_days
+  puts "It took #{days} days to have three happy days"
+end
```

Comment:
Hmm.... ok. This works. But it doesn't follow what was asked. I don't want to have to call analyze_mood in order to find out how many days it takes to get 3 happy days and I don't want the number of happy_days to print every time I call analyze_mood. You solve both problems, which is great! But I don't want these two things tied together. I want to be able to do either operation independent of the other.